canonical/design.ubuntu.com
# WD-1299 use new vanilla layout on design ubuntu com site

## Done

- Use the new layout on the following pages:
  - Home
  - Accessibility
  - Contribute
  - Font
  - Resources
  - 404
- Minor fixes on the resources page
  - Remove the Illustration section
  - Make the Icon set section dark mode switch hide on medium and small screens

## QA

- Open the site and inspect the pages for the new layout
- Navigate to `/resources` 
- Check if the icon set switch hides on medium and small screens
- Check that the illustration section has been removed


## Issue / Card

- [WD-1299](https://warthogs.atlassian.net/browse/WD-1299)
- [WD-1297](https://warthogs.atlassian.net/browse/WD-1297)


[WD-1299]: https://warthogs.atlassian.net/browse/WD-1299?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[WD-1297]: https://warthogs.atlassian.net/browse/WD-1297?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
